pandas/core/generic.py

```diff
@@ -4871,7 +4872,7 @@ def describe_1d(data, percentiles):
                     names.append(name)
         d = pd.concat(ldesc, join_axes=pd.Index([names]), axis=1)
         d.columns.names = data.columns.names
-        return d
+        return self._constructor(d)
```

Comment:
Actually, I had already added the logic to concat, so the constructor was unnecessary here. I have removed it.